### ISSUE TITLE
Switch the default `stacks` list to the wildcard stack

### ISSUE THREAD

**Issue by edmorley** [heroku/cnb-shim#73]
Currently if the URL doesn't specify a stacks list via the `stacks` parameter, `cnb-shim` uses a default value like:
https://github.com/heroku/cnb-shim/blob/81f6a88f9678ece00b42c1994cfd299cca3851ee/server/main.go#L85

This default value is:
- somewhat of a guess, since there is no way that cnb-shim can know what stacks the buildpack is actually compatible with
- then something we have to continually keep up to date (eg #42 and #72)

We should consider switching it to the wildcard/any stack, `*`, which permits all stacks.

However, the any stack feature requires Buildpack API 0.5+:
https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5

...but cnb-shim is still using Buildpack API 0.4, so switching the default stacks list is blocked on #69.

**Comment by edmorley** [comment]
Wontfix is cnb-shim is EOL.
